src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -1041,25 +1147,26 @@ def resize_token_embeddings(self, new_num_tokens):
             num_tokens_to_copy = min(self.final_logits_bias.shape[0], new_num_tokens)
             init_bias = tf.zeros((new_num_tokens,))
             init_bias[:num_tokens_to_copy] = self.final_logits_bias.value()[:num_tokens_to_copy]
```

Comment:
There is an issue here. My bad.
```suggestion
            init_bias = np.zeros((1, new_num_tokens))
            init_bias[:num_tokens_to_copy] = self.final_logits_bias.value()[:, :num_tokens_to_copy]
```

Reply:
do you want me to change it here?

Reply:
Well, I just realized that there are two other models where there is `tf.zeros` instead of `np.zeros` so I will handle this myself in a PR once you merged this one 👍